source/components/Tokens/index.jsx

```diff
@@ -20,8 +19,8 @@ const Tokens = () => {
   const [loading, setLoading] = useState(true);
   const dispatch = useDispatch();
   const icpPrice = useICPPrice();
-  const { navigator } = useRouter();
   const { onScroll, fullScroll } = useScroll();
+  const [activateButton, setActivateButton] = useState(false);
```

Comment:
We should probably change this to `displayCustomToken` or something more descriptive